server.go

```diff
@@ -15,6 +17,16 @@ func main() {
 		s.Engine.Use(server.RequestLoggerHandler)
 	}
 
+	s.Engine.Use(cors.Middleware(cors.Config{
+		Origins:         "*",
+		Methods:         "GET, PUT, POST, DELETE",
+		RequestHeaders:  "Origin, Authorization, Content-Type",
```

Comment:
Consider adding "If-Match" and "If-None-Exist". I don't think we support these to do conditional updates or creates, but just something to keep in mind.